src/tools/cephfs/first-damage.py

```diff
@@ -56,7 +56,7 @@
 REMOVE = False
 POOL = None
 NEXT_SNAP = None
-CONF = os.environ.get('CEPH_CONF')
+CONF = os.environ.get('CEPH_CONF', '/etc/ceph/ceph.conf')
```

Comment:
my recollection is that this should use that as the default if not set. But I guess not since you submitted this PR...